src/app/documents/components/document-view/document-view.component.ts

```diff
@@ -61,4 +65,11 @@ export class DocumentViewComponent extends BaseComponent implements OnInit, OnCh
   public onNodeEditButtonClick(node: DocumentNode) {
     this.editNodeModal.open(node);
   }
+
+  private isDocumentInConsultation(documentConfiguration: DocumentConfiguration): boolean {
+    const now = new Date().getTime();
```

Comment:
Date().getTime() returns current date _and time_. If it's past 0AM, `now <= consultationEndDate` will return false even both times belong to the same day.
```suggestion
    const now = new Date().getHours(0,0,0,0);
```